scripts/flattener.py

```diff
@@ -798,6 +798,31 @@ def map_cell_enrichment(row):
 	else:
 		glob.cxg_obs['library_sequencing_run'] = 'unknown'
 
+# Sanity check that all prefixes/suffixes are actually in ProcMatrixFile
+def prefixes_suffixes_presence_and_duplicate_check(glob):
+	mfinal_cell_identifiers = glob.mfinal_adata.obs.index.to_list()
```

Comment:
`mfinal_cell_identifiers` outside of this function is defined on line 878. Might be good for larger datasets to pass the previously defined `mfinal_cell_identifiers` to this function so that the list doesn't have to be created again.